assets/src/edit-story/components/library/panes/text/textSets/textSetsPane.js

```diff
@@ -139,12 +140,26 @@ function TextSetsPane({ paneRef }) {
     [textSets]
   );
 
-  const handleSelectedCategory = useCallback((selectedCategory) => {
-    setSelectedCat(selectedCategory);
-    localStore.setItemByKey(`${LOCAL_STORAGE_PREFIX.TEXT_SET_SETTINGS}`, {
-      selectedCategory,
-    });
-  }, []);
+  const speak = useLiveRegion('polite');
+
+  const handleSelectedCategory = useCallback(
+    (selectedCategory) => {
+      setSelectedCat(selectedCategory);
+      speak(
+        selectedCategory === null
+          ? __(`Deselected text set filter`, 'web-stories')
```

Comment:
```suggestion
          ? __('Deselected text set filter', 'web-stories')
```

Reply:
🤔 This is good. What do you think about making this be something like 'show all' or 'no filter applied'? I don't know which way I'm leaning for the announcement. I see two options:
1. The removal of a filter
2. How many text sets are being displayed

Not sure what is easier to understand, but I think what you have now is totally good too

Reply:
I think I like `Show all text sets` best